### PR TITLE
Hopefully fixed log4j exploit(I am not the best build system wizard)for 1.17 fabric

### PR DESCRIPTION
**Pull request by MaxBr5**
<pre>I thought I fixed it until I did "gradlew dependencies" so I naturally did
more fixes 'till gradlew dependencies gave me more optimistic results.
credits:
https://stackoverflow.com/questions/70317385/gradle-java-how-to-upgrade-log4j-safely
https://blog.gradle.org/log4j-vulnerability</pre>

### CODE REVIEW

**Comment by MaxBr5** [comment]
<pre>I have proven that I am indeed an amateur with the previous request.</pre>

File: build.gradle [hunk 1]
```diff
@@ -18,11 +18,28 @@ repositories {
 	// for more information about repositories.
 }
 
-minecraft {
```
Comment (FionaTheMortal) [inline build.gradle:21]: Why are you making this change over here?

Comment (magneticflux-) [reply build.gradle:21]: Not the author, but the `minecraft` scope is deprecated in Loom 0.10

Comment (FionaTheMortal) [reply build.gradle:21]: Thanks :)

**Pull request merged**